modules/light-clients/08-wasm/testing/mock_engine.go

```diff
@@ -122,15 +122,15 @@ type MockWasmEngine struct {
 // StoreCode implements the WasmEngine interface.
 func (m *MockWasmEngine) StoreCode(code wasmvm.WasmCode) (wasmvm.Checksum, error) {
 	if m.StoreCodeFn == nil {
-		panic("mock engine is not properly initialized")
+		panic("mock engine is not properly initialized: StoreCodeFn is nil")
```

Comment:
since we are modifying it already, a mega nit would be to change these to errors, I think we try and panic on errors in almost all places where we have panics.